exercises/diffie-hellman/example.js

```diff
@@ -0,0 +1,29 @@
+export default class DiffieHellman {
+
+  constructor(p, g) {
+    if (p < 2 || g < 2 || p > 7919 || g > 7919) {
+      throw Error('Constructor arguments must be between 2 and 7920!');
+    }
+    if (!PRIMES.includes(p) && PRIMES.includes(g)) {
```

Comment:
This and previous `if` statements could be extracted to a function. That function might throw errors, and the constructor will be like:

```
constructor(p, g) {
  validate(p);
  validate(g); // or validate(p, g), whatever
  this.p = p;
  this.g = g;
}
```